win32/tests/lazy_init_ut - Copy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+#Copyright (c) Microsoft. All rights reserved.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+compileAsC11()
+set(theseTestsName lazy_init_ut)
```

Comment:
>lazy_init_ut [](start = 19, length = 12)

ha a "lazy_init_ut - Copy" got added to the PR

Reply:
a major copy&paste! now deleted away

---
In reply to: [466044647](https://github.com/Azure/azure-c-pal/pull/28#discussion_r466044647) [](ancestors = 466044647)